### PR TITLE
Linking error potential fix

### PR DESCRIPTION
**Pull request by neotheprogramist**
## Description
This pull request addresses a critical linking error encountered in the Hyperledger Besu native library. The issue came to light following a modification in the method invocation within the `PedersenHasher` class, where `LibIpaMultipoint.commit` was changed to `LibIpaMultipoint.commit_root`. As a result of this change, a `java.lang.UnsatisfiedLinkError` is triggered when calling the `commit_root` method, with the error message: `'byte[] org.hyperledger.besu.nativelib.ipamultipoint.LibIpaMultipoint.commit_root(byte[])'`.

### Proposed Fix
The fix proposed in this pull request involves renaming the `commit_root` method from snake_case to camelCase. This renaming aims to:

- Align with Java's standard naming conventions.
- Potentially resolve the linking issues that are currently being faced.

### Context
While other methods in the `LibIpaMultipoint` class, such as `commit` and `pedersenHash`, continue to operate as expected, it is believed that the naming convention employed for `commit_root` may be contributing to the linking error.

### CODE REVIEW

**Comment by matkt** [comment]
seems to work . could we add a small description in the PR ?

**Comment by thomas-quadratic** [comment]
Shouldn't we have a test in ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/ for this method as well ?

**Comment by thomas-quadratic** [comment]
Good with me. Thanks!

File: ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/CommitRootTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package org.hyperledger.besu.nativelib.ipa_multipoint;
+
```
Comment (matkt) [inline ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/CommitRootTest.java:2]: Seems good, could you just add a header like this file https://github.com/hyperledger/besu-native/blob/main/gnark/src/main/java/org/hyperledger/besu/nativelib/gnark/LibGnark.java#L1

**Review by matkt — APPROVED** [review]
Lgtm

**Comment by thomas-quadratic** [comment]
Good for me as well, thanks @neotheprogramist

**Pull request merged**